.github/workflows/linux-checks.yaml

```diff
@@ -64,10 +70,19 @@ jobs:
         uses: arduino/setup-protoc@master
       - name: Add bin to path
         run: echo "::add-path::${{env.GOPATH}}/bin"
+      - name: Go mod cache setup
+        uses: actions/cache@v2
+        with:
+          path: ${{env.GOPATH}}/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
       - name: Fetch vendor and check succesful build
         run: |
           go mod download
-          make vendor install-build-tools build
+          make vendor
+          go install -v github.com/cyraxred/go-acc
```

Comment:
WTF? Why it is not in our dependencies?

Reply:
It is, but we don't need here all the build tools, just go-acc

Reply:
switched to common install-build-tools